base/main.cpp

```diff
@@ -153,24 +153,24 @@ static Common::Error runGame(const EnginePlugin *plugin, OSystem &system, const
 	// needed because otherwise the g_system->getSupportedFormats might return
 	// bad values.
 	g_system->beginGFXTransaction();
-		g_system->setGraphicsMode(ConfMan.get("gfx_mode").c_str());
```

Comment:
This is done on purpose, to visually set up code which is within the transaction.